.github/workflows/merge.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and release
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  node:
+    name: Node 12
+    # buster distribution has git >v2.17 required for actions/checkout@v2
+    container: node:12-buster
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Sync packages with cache
+        uses: actions/cache@v1
+        with:
+          path: ./node_modules
+          key: node:12-buster-node_modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: node:12-buster-node_modules-
+
+      - name: Install packages
+        run: yarn install --frozen-lockfile
+
+      - name: Build the docs
+        run: yarn document
+
+      - name: Compile src
+        run: yarn build
+
+      - name: Commit docs and dist
```

Comment:
Are we still committing the dist folder?

Reply:
No, I forgot to change the name. I saw  this later and updated it in the other PR #19

Reply:
Okay for me